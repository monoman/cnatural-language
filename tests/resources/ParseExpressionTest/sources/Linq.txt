from s in names
select s.ToUpper()