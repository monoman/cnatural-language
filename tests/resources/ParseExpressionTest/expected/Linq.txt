from s in Name(names)
select (Name(s)).ToUpper()